tests/xtro-sharpie/common-Vision.ignore

```diff
@@ -45,3 +51,77 @@
 !missing-selector! VNRequest::setRevision: not bound
 ## init is available but it's abstract (type) and not a designated initializer (so we don't bind it)
 !missing-selector! VNRequest::init not bound
+
+# Sharpie needs a fix to only get simd_float2 intead of const simd_float2
+!unknown-simd-type-mapping! The Simd type NS_RETURNS_INNER_POINTER const simd_float2 * does not have a mapping to a managed type. Please add one in SimdCheck.cs
```

Comment:
Can't this be fixed here (by adding a case for `NS_RETURNS_INNER_POINTER`)? https://github.com/xamarin/xamarin-macios/blob/3a0580750d34601607461b52304c3d6532d6c0e5/tests/xtro-sharpie/SimdCheck.cs#L244-L271

Reply:
🤔 I'll have a look maybe it makes sense to fix there now